File: data/evaluation/geeks_for_geeks_successful_test_scripts/java/FIND_SUM_EVEN_INDEX_BINOMIAL_COEFFICIENTS_1.java
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//

import java.util. *;
import java.util.stream.*;
import java.lang.*;
import javafx.util.Pair;
public class FIND_SUM_EVEN_INDEX_BINOMIAL_COEFFICIENTS_1{
static int f_gold ( int n ) {
  return ( 1 << ( n - 1 ) ) ;
}


//TOFILL

public static void main(String args[]) {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(56);
    param0.add(28);
    param0.add(4);
    param0.add(24);
    param0.add(72);
    param0.add(30);
    param0.add(48);
    param0.add(32);
    param0.add(13);
    param0.add(19);
    for(int i = 0; i < param0.size(); ++i)
    {
        if(f_filled(param0.get(i)) == f_gold(param0.get(i)))
        {
            n_success+=1;
        }
    }
    System.out.println("#Results:" + n_success + ", " + param0.size());
}
}

File: data/evaluation/geeks_for_geeks_successful_test_scripts/python/FIND_SUM_EVEN_INDEX_BINOMIAL_COEFFICIENTS_1.py
# Copyright (c) 2019-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#
def f_gold ( n ) :
    return ( 1 << ( n - 1 ) )


#TOFILL

if __name__ == '__main__':
    param = [
    (56,),
    (28,),
    (4,),
    (24,),
    (72,),
    (30,),
    (48,),
    (32,),
    (13,),
    (19,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        if f_filled(*parameters_set) == f_gold(*parameters_set):
            n_success+=1
    print("#Results: %i, %i" % (n_success, len(param)))